src/components/__tests__/IspSelectWithIncidents-test.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import chai, { expect } from 'chai';
+
+import chaiEnzyme from 'chai-enzyme';
+import { IspSelectWithIncidents } from 'components';
+
+chai.use(chaiEnzyme());
+
+describe('components', () => {
+  describe('IspSelectWithIncidents', () => {
+    it('selecting multiple ISPs', () => {
+
+      let selected = [];
+
+      const mockChange = function(ispId) { selected.push(ispId); };
```

Comment:
Since you also have another function (on line 60) that's also supposed to resemble what dispatch does, I think you should combine the logic and create one function that you can just call in each test.